src/much-selector.js

```diff
@@ -256,6 +266,22 @@ class MuchSelector extends HTMLElement {
     this._app.ports.disableChangedReceiver.send(this._disabled);
   }
 
+  get maxdropdownitems() {
+    return this._maxdropdownitems;
+  }
+
+  set maxdropdownitems(value) {
```

Comment:
We should cast this value to a number... and if it can't be a number we should throw a `TypeError`